mycli/packages/prompt_utils.py

```diff
@@ -7,6 +7,26 @@
 from .parseutils import is_destructive
 
 
+class ConfirmBoolParamType(click.ParamType):
+    name = 'confirmation'
+
+    def convert(self, value, param, ctx):
+        if isinstance(value, bool):
+            return bool(value)
```

Comment:
When would we get a boolean value?

Reply:
hmm, probably never. I copied it from [here](https://github.com/pallets/click/blob/2c622eed402c64e1645db3904c2410f6c05fbf19/click/types.py#L348) without much thought, maybe we can delete this conditional branch.